File: Facturacion/Models/Productos.cs
using System;
using System.Collections.Generic;

namespace Facturacion.Models
{
    public partial class Productos
    {
        public int IdProductos { get; set; }
        public int IdFabricante { get; set; }
        public string Nombre { get; set; }
        public decimal Precio { get; set; }
        public string Descripcion { get; set; }
        public string Estado { get; set; }
        public int Stock { get; set; }
        public string Modelo { get; set; }

        public Fabricantes IdFabricanteNavigation { get; set; }
    }
}

File: src/app/pages/Models/productos.ts
export class Productos {

  constructor(
    public  IdProductos:number = 0,
    public  IdFabricante:number = 0,
    public  Nombre:string = '',
    public  Precio:number = 0,
    public  Descripcion:string = '',
    public  Estado:string = '',
    public  Stock:number = 0,
    public  Modelo :string = '',
    public UrlPhoto: string = ''
  ) { }

}
